src/gridfs/mod.rs

```diff
@@ -14,10 +14,10 @@
 //! which you want access without having to load the entire file into memory.
 //!
 //! ```no_run
-//! # use mongodb::{Client, ThreadedClient};
+//! # use mongodb::{Connector, ThreadedClient};
```

Comment:
This example was updated so that the tests would pass. All other changes to this file are from rustfmt.